allensdk/brain_observatory/behavior/data_objects/metadata/behavior_metadata/behavior_metadata.py

```diff
@@ -321,6 +335,12 @@ def behavior_session_uuid(self) -> Optional[uuid.UUID]:
     def behavior_session_id(self) -> int:
         return self._behavior_session_id.value
 
+    @property
+    def project_code(self) -> Optional[str]:
```

Comment:
According to the `ProjectCode` class, it looks like it always sets project code to a value, so I'm confused by the checking for None and the Optional type. The Optional type is specifically for `None`

Reply:
Removed.